tests/FSharp.Test.Utilities/Compiler.fs

```diff
@@ -464,6 +464,9 @@ module rec Compiler =
         | FS fs -> FS { fs with IgnoreWarnings = true }
         | _ -> failwith "TODO: Implement ignorewarnings for the rest."
 
+    let withCulture culture (cUnit: CompilationUnit) : CompilationUnit =
```

Comment:
It's a good helper, we can also, probably add `--preferreduilang:en-US` to the `defaultOptions`, so we, by default run in the `en-US` culture, and if we need a different one, we just use `withCulture`.